### ISSUE TITLE
libmodsecurity parameter not recognized

### ISSUE THREAD

**Issue by Semirben** [SpiderLabs/ModSecurity-apache#28]
Hi everyone,

I'm compiling the recent ModSecurity V3 with the Apache connector. ModSecurity is compiled with these options
```
./configure --prefix=/opt/modsecurity --enable-parser-generation --without-lua
```
My Apache (2.4.29) is located at `/opt/apache/`
So my configure arguments for the connector are
```
./configure --prefix=/opt/modsecurity --with-apxs=/opt/apache-2.4.29/bin/apxs --with-apache=/opt/apache-2.4.29/bin/httpd --with-libmodsecurity=/opt/modsecurity/
```
but somehow this parameter is not recognized:
```
checking for a BSD-compatible install... /usr/bin/ginstall -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... /usr/bin/gmkdir -p
checking for gawk... gawk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
configure: looking for Apache module support via DSO through APXS
configure: found APXS at /opt/apache-2.4.29/bin/apxs
configure: looking for libmodsecurity
configure: error: couldn't find libmodsecurity
```

is this a known error? My config.log is attached.

Thanks and regards,
Daniel
[config.log](https://github.com/SpiderLabs/ModSecurity-apache/files/1652500/config.log)

**Comment by JRemitz** [comment]
I just noticed the same thing.  It looks like V3LIB in the configure file is not set.  I worked around it with the following update to `configure` around line 2443 by setting V3LIB when overriding V3PATH:
```# Check whether --with-libmodsecurity was given.
if test "${with_libmodsecurity+set}" = set; then :
  withval=$with_libmodsecurity;
  if test "$withval" = "yes"; then
    V3PATH=/usr/local/modsecurity/
  else
    V3PATH="$withval"
    V3LIB="$withval/lib/"
  fi

fi
```
I don't know if that's the best way or if more tests need to happen, but it worked just fine for me afterwards.

**Comment by JRemitz** [comment]
PR addressed configure issue however it's still not "making" anything for me.  make and make install complete with a lot of warnings and: 
```
make[1]: Nothing to be done for `install-exec-am'.
make[1]: Nothing to be done for `install-data-am'.
```

**Comment by zimmerle** [comment]
fixed as #29.
